Sources/Purchasing/StoreKitAbstractions/StoreProductDiscount.swift

```diff
@@ -29,6 +29,13 @@ public typealias SK2ProductDiscount = StoreKit.Product.SubscriptionOffer
 @objc(RCStoreProductDiscount)
 public final class StoreProductDiscount: NSObject, StoreProductDiscountType {
 
+    private static let decimalRoundingBehavior = NSDecimalNumberHandler(roundingMode: .up,
+                                                                        scale: 2,
+                                                                        raiseOnExactness: false,
+                                                                        raiseOnOverflow: false,
+                                                                        raiseOnUnderflow: false,
+                                                                        raiseOnDivideByZero: false)
```

Comment:
yeah, not exactly clean to have this duplicated for both classes, I was going for "quick PR to show off a possible solution"